src/PROPOSAL/detail/PROPOSAL/crosssection/parametrization/Compton.cxx

```diff
@@ -11,7 +11,7 @@ using namespace PROPOSAL;
 double crosssection::Compton::GetLowerEnergyLim(
     const ParticleDef&) const noexcept
 {
-    return 2. * ME;
+    return pow(ALPHA, 2) * ME / 2; // ionization energy of K-shell electron for hydrogen
```

Comment:
compared to the lower limit of the photo effect parametrization of $(Z\alpha)^2 m_e/2$ this now ensures that the Compton lower limit is always at least as low as the photo effect.